train_.py

```
@@ -0,0 +1,183 @@
+pdb_protBERT_dataset_test = datasets['test']
+pdb_protBERT_dataset_valid = datasets['valid']
+
+print(f"Loaded datasets: Train={len(pdb_protBERT_dataset_train)}, Test={len(pdb_protBERT_dataset_test)}, Valid={len(pdb_protBERT_dataset_valid)}")
+
+# Constants
+BATCH_SIZE = 128
+EPOCHS = 1000
+LEARNING_RATE = 0.0001  
+THRESHOLD = 0.50
+
+BEST_MODEL_PATH = f'/home/hpc_users/2019s17273@stu.cmb.ac.lk/ganeshiny/protein-go-predictor/model_and_weight_files/WEIGHTS_2LAYERS.pth'
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)